packaging/installer/methods/kickstart.md

```diff
@@ -5,181 +5,130 @@ import TabItem from '@theme/TabItem';
 
 # Install Netdata with kickstart.sh
 
-![last hour badge](https://registry.my-netdata.io/api/v1/badge.svg?chart=web_log_nginx.requests_by_url_pattern&options=unaligned&dimensions=kickstart&group=sum&after=-3600&label=last+hour&units=kickstart%20downloads&precision=0) ![today badge](https://registry.my-netdata.io/api/v1/badge.svg?chart=web_log_nginx.requests_by_url_pattern&options=unaligned&dimensions=kickstart&group=sum&after=-86400&label=today&units=kickstart%20downloads&precision=0)
+`kickstart.sh` is the recommended way to install Netdata.
 
-**`kickstart.sh` is the recommended way of installing Netdata.**
+This installation script works on all major Linux distributions. It automatically detects the best way to install Netdata for your system.
 
-This script works on all Linux distributions, by detecting the optimal method of installing Netdata directly to the operating system.
+<details><summary>🧩 What does kickstart.sh actually do?</summary>
 
+1. Detects your OS and environment
+2. Checks for an existing Netdata installation
+3. Installs using:
+    - Native packages (preferred)
+    - Static build (fallback)
+    - Build from source (last resort)
+4. Installs an auto-update cron job (unless disabled)
+5. Optionally connects your node to Netdata Cloud
 
-## Installation
+</details>  
 
-> **Tip**
->
-> If you are unsure whether you want nightly or stable releases, read the [related section](/docs/netdata-agent/versions-and-platforms.md) of our Documentation, detailing the pros and cons of each release type.
+---
 
-To install Netdata, run the following as your normal user:
+## Quick Overview
+
+| Task                  | Command / Location             | Notes                                   |
+|-----------------------|--------------------------------|-----------------------------------------|
+| Install Netdata       | Run `kickstart.sh`             | Choose nightly or stable release        |
+| Connect to Cloud      | Use claim token                | Connect node to Netdata Cloud           |
+| Customize install     | Pass flags to control behavior | Directory, release, update control      |
+| Export config for IaC | Copy config from Cloud UI      | For automation & Infrastructure as Code |
+
+---
+
+## Run the One-Line Install Command
+
+To install and connect to Netdata Cloud in a single step from your terminal:
 
 <Tabs>
   <TabItem value="wget" label="wget">
 
-  <OneLineInstallWget/>
+<OneLineInstallWget/>
 
   </TabItem>
   <TabItem value="curl" label="curl">
 
-  <OneLineInstallCurl/>
+<OneLineInstallCurl/>
```

Comment:
why change indentation here? I don't know if this will work or not, w/o the indent